_data/team-members.yml

```diff
@@ -59,8 +59,10 @@ members:
     projects:
        - title: scalafix
          percentage: 50
+       - title: buildTools
+         percentage: 20
```

Comment:
I spent some time last week helping out with bsp and I've been sharing experience with sbt server. Macros have received less time mostly because I'm waiting for dotty implementation to catch up with my changes, there have been no blocking issues on my front there for almost 7 weeks now.